app/views/forgotPassword.scala.html

```diff
@@ -0,0 +1,18 @@
+@(forgotPasswordForm: Form[String])(implicit request: RequestHeader, lang: Lang)
+@import views.html.bootstrap._
+
+@main("Project Sidewalk - Recover Password") {
+    @navbar(None)
+    <fieldset class="col-md-6 col-md-offset-3" style="margin-top:51px">
+        <legend>@Messages("forgot.password.title")</legend>
+        @helper.form(action = routes.ForgotPasswordController.submit(), 'autocomplete -> "off") {
```

Comment:
Do we need to turn off autocomplete for the email address?

Reply:
Probably not? I can't think of any safety issues